app/public/redirect.html

```diff
@@ -1,7 +1,113 @@
 <html>
-  <head></head>
+  <head>
+    <style>
+      * {
+        box-sizing:border-box;
+      }
+
+      html, body{
+        background: #006BE8;
+        height:100%;
+        padding:0;
+        margin:0;
+      }
+
+      .container {
+        width:100%;
+        height:100%;
+        
+        display:flex;
+        justify-content:center;
+        align-items:center;
+        flex-direction: column;
+      }
+      
+      h1.title {
+        font-size: 32px;
+        color: #FFFFFF;
+        font-family: 'Roboto', sans-serif !important;
+        margin: 0;
+        text-align: center;
+      }
+      p.info {
+        font-size: 24px;
+        color: #FFFFFF;
+        font-family: 'Roboto', sans-serif !important;
+        margin: 24px 0 40px;
+      }
+
+      .spinner .beat {
+        background-color: #ffffff;
+        height: 12px;
+        width: 12px;
+        margin: 2px;
+        border-radius: 100%;
+        -webkit-animation: beatStretchDelay 0.7s infinite linear;
+        animation: beatStretchDelay 0.7s infinite linear;
+        -webkit-animation-fill-mode: both;
+        animation-fill-mode: both;
+        display: inline-block;
+      }
+      .spinner .beat-odd {
+        animation-delay: 0s;
+      }
+      .spinner .beat-even {
+        animation-delay: 0.35s;
+      }
+      @-webkit-keyframes beatStretchDelay {
+        50% {
+          -webkit-transform: scale(0.75);
+          transform: scale(0.75);
+          -webkit-opacity: 0.2;
+          opacity: 0.2;
+        }
+        100% {
+          -webkit-transform: scale(1);
+          transform: scale(1);
+          -webkit-opacity: 1;
+          opacity: 1;
+        }
+      }
+      @keyframes beatStretchDelay {
+        50% {
+          -webkit-transform: scale(0.75);
+          transform: scale(0.75);
+          -webkit-opacity: 0.2;
+          opacity: 0.2;
+        }
+        100% {
+          -webkit-transform: scale(1);
+          transform: scale(1);
+          -webkit-opacity: 1;
+          opacity: 1;
+        }
+      }
+
+      @media (min-width: 768px) {
+        h1.title {
+          font-size: 40px;
+        }
+        p.info {
+          font-size: 28px;
+        }
+
+        .spinner .beat {
+          height: 15px;
+          width: 15px;
+        }
+      }
+    </style>
+  </head>
   <body>
-    <div>Redirecting you back to Torus app...</div>
+    <div class="container">
+      <h1 class="title content">You are being redirected to payment provider</h1>
```

Comment:
please change copywriting to 'You are being redirected'

Reply:
> please change copywriting to 'You are being redirected'

Done